spec/unit/provider/template/content_spec.rb

```diff
@@ -20,6 +20,14 @@
 
 describe Chef::Provider::Template::Content do
 
+  let(:enclosing_directory) {
+    canonicalize_path(File.expand_path(File.join(CHEF_SPEC_DATA, "cookbooks")))
+  }
+
+  let(:resource_path) {
+    canonicalize_path(File.expand_path(File.join(enclosing_directory, "openldap/templates/default/openldap_stuff.conf.erb")))
+  }
```

Comment:
enclosing_directory should probably be a tmpdir that you create.  then resource_path should just be "openldap_stuff.conf" (in that directory).

I think you're asking to overwrite your source with your destination here which probably only works because its all stubbed out (but still doesn't make sense and makes for a really brittle and confusing test).